packit_service/worker/jobs.py

```diff
@@ -271,6 +279,13 @@ def process_comment_jobs(
                 success=True, details={"msg": "Account is not whitelisted!"}
             )
 
+        # VERY UGLY
+        # TODO: REFACTOR !!!
+        if handler_kls == GitHubPullRequestCommentCoprBuildHandler and isinstance(
+            event, PullRequestCommentPagureEvent
+        ):
+            handler_kls = PagurePullRequestCommentCoprBuildHandler
+
```

Comment:
this is very ugly, but enough for my testing and probably can stay here till refactoring.
any suggestions are welcome

Reply:
Some nicer way would require lists in `MAP_COMMENT_ACTION_TO_HANDLER` and making some logic on top of it.

I would use this approach for now and discuss after summit -- I'll add a note to our refactor agenda about this.

Reply:
I think we already discuss that, but do we want to use whitelist for centos-stream? Can we move this above the whitelisting code?

Reply:
yes u are right, will fix it

Reply:
fixed  - moved logic to Whitelist to avoid whitelisting